WTWSMS/events/ACR_misc_hellenic_flavor.txt

```diff
@@ -5419,6 +5438,7 @@ character_event = {
 		NOT = { has_character_flag = VIET_artemis_sex_question }
 		NOT = { trait = celibate }
 		NOT = { trait = eunuch }
+		supernatural_events_trigger = yes
 	}
 	
 	weight_multiplier = {
```

Comment:
How many people haven't spoken to gods and prophets? This isn't more supernatural than a dream or lunacy. People believe that they meet deities, and while I'm an agnostic, it is not because they have supernatural powers.